### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by hmzisb**

### CODE REVIEW

File: src/components/we-are-using/index.js [hunk 1]
```diff
@@ -258,6 +258,11 @@ const companies = [
 		href: 'https://sqlframes.com/',
 		logo: 'sqlframes.svg'
 	}
+	{
+		name: 'BestForAndroid',
```
Comment (binyamin) [inline src/components/we-are-using/index.js:262]:
```suggestion
	},
	{
		name: 'BestForAndroid',
```

File: src/components/we-are-using/index.js [hunk 2]
```diff
@@ -258,6 +258,11 @@ const companies = [
...
+		href: 'https://bestforandroid.com/',
+		logo: 'https://bestforandroid.com/wp-content/uploads/2020/03/cropped-bestforandroid-1.png'
+	}
```
Comment (binyamin) [inline src/components/we-are-using/index.js:265]: Please place the image in the `src/assets/we-are-using/` folder.

**Review by binyamin — CHANGES_REQUESTED** [review]

**Comment by binyamin** [comment]
@hmzisb This is a Wordpress site. What does this have to do with Preact?

**Review by marvinhagemeister — CHANGES_REQUESTED** [review]
I'm afraid we have to close this PR as the site contains links to circumvent licensing of android apps, which is not legal in many countries. That's not something we want to be associated with for legal reasons.

**Pull request closed without merging**